qutebrowser/browser/commands.py

```diff
@@ -832,8 +832,9 @@ def paste(self, sel=False, tab=False, bg=False, window=False):
             self._open(url, tab, bg, window)
 
     @cmdutils.register(instance='command-dispatcher', scope='window',
-                       count='count')
-    def tab_focus(self, index: {'type': (int, 'last')}=None, count=None):
+                       name=['tab-focus', 'b'], count='count',
```

Comment:
FWIW I believe the default keybinding in pentadactyl puts this functionality under the `b` keybinding too, so it's probably not completely unintuitive, at least for those coming from pentadactyl. However, I also agree with leaving people to do their own thing.